loadbalancer/zk/client.go

```diff
@@ -0,0 +1,82 @@
+package zk
+
+import (
+	"github.com/go-kit/kit/log"
+	"github.com/samuel/go-zookeeper/zk"
+	"strings"
+	"time"
+)
+
+const ()
+
+// Client is a wrapper arround the zookeeper client.
+type Client interface {
+	// GetEntries queries the path in zookeeper, places a watch on it
+	// and retrieves data from its current child nodes
+	GetEntries(path string) ([]string, <-chan zk.Event, error)
+	// createParentNodes tries to create the path
+	// in case it does not exist yet on Zookeeper
+	createParentNodes(path string) error
+}
```

Comment:
Refactoring mistake, The method should have been exported. Thinking about it, I'm not entirely sure if it should have an interface at all but will keep that in so people are allowed to provide an alternative ZooKeeper client implementation in case it suits them better. Given that some of the other loadbalancer implementations have exported Clients too I'll keep it in.